src/components/policy/policy_regular/src/sql_pt_representation.cc

```diff
@@ -476,10 +476,9 @@ void SQLPTRepresentation::GatherModuleMeta(
   LOG4CXX_INFO(logger_, "Gather Module Meta Info");
   utils::dbms::SQLQuery query(db());
   if (query.Prepare(sql_pt::kSelectModuleMeta) && query.Next()) {
-    *meta->ccpu_version = query.GetString(0);
```

Comment:
@ydementieiev Why do you create PT with ccpu_version but removed possibility to get it from there?

Reply:
@AByzhynar you are right, it was fall, thank you !!,  [added here](https://github.com/LuxoftSDL/sdl_core/pull/28/commits/62bc6f3dc160da972e4eecbcc36f172bb64a92be)